components/bootloader_support/src/flash_encryption/flash_encrypt.c

```diff
@@ -311,7 +311,11 @@ esp_err_t esp_flash_encrypt_enable(void)
 #endif
     }
 
+#if CONFIG_EFUSE_VIRTUAL
+    ESP_LOGW(TAG, "Flash encryption not really completed. Must disable virtual efuses");
```

Comment:
This will probably fail the tests cases from efuse example:

https://github.com/espressif/esp-idf/blob/edd815af2e7d541b25ff3a74c59a7fe0612df42d/examples/system/efuse/pytest_system_efuse_example.py#L243

Reply:
Solution: just log both messages?
```
    ESP_LOGI(TAG, "Flash encryption completed');

#if CONFIG_EFUSE_VIRTUAL
    ESP_LOGW(TAG, "Flash encryption not really completed. Must disable virtual efuses");
#endif
```

Reply:
I think the clean solution would be to now allow `CONFIG_EFUSE_VIRTUAL` for release mode of the security features through Kconfig dependency itself. But we have some test configs that depend on this scenario, we may have to refactor those. I will check it further.

Reply:
I think the both logs solution is actually better. It shows that the normal operations have "completed"